python/llm/src/ipex_llm/transformers/utils.py

```diff
@@ -361,3 +361,15 @@ def get_modelscope_hf_config(model_id_or_path: str,
 def is_torch_bf16_gpu_available():
     # always true for XPU and CPU
     return True
+
+
+# Arc platfrom does not support FP64,
+# Disable FP64 in DeepSpeedZeroOptimizer_Stage3's _constant_buffered_norm2  method
```

Comment:
Add link to https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/runtime/zero/stage3.py#L1365-L1372

Reply:
What's different between our implementation and ds's one?

Reply:
ds is double(), fp64
here removes double(), as Arc does not support fp64

Reply:
OK